templates/pulp.conf.erb

```diff
@@ -38,6 +42,14 @@ WSGIImportScript /srv/pulp/webservices.wsgi process-group=pulp application-group
     SSLVerifyClient optional
 </Files>
 
+Alias /pulp/static /var/lib/pulp/static
+
+<Location /pulp/static>
+    SSLRequireSSL
+    Options +Indexes
+    Require all granted
```

Comment:
I think you need to account for differences in apache versions: 

https://github.com/Katello/puppet-katello/blob/master/templates/etc/httpd/conf.d/05-foreman.d/katello.conf.erb#L11-L15